hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/snapshot/ExportSnapshot.java

```diff
@@ -769,12 +831,6 @@ public int compare(Pair<SnapshotFileInfo, Long> a, Pair<SnapshotFileInfo, Long>
       }
     }
 
-    if (LOG.isDebugEnabled()) {
-      for (int i = 0; i < sizeGroups.length; ++i) {
-        LOG.debug("export split=" + i + " size=" + Strings.humanReadableInt(sizeGroups[i]));
-      }
-    }
-
```

Comment:
I took this log out because it makes sense now that getBalancedSplits() can be run multiple times, plus I added new logging that supplants it.